client/src/pages/Course.js

```diff
@@ -1,9 +1,8 @@
 import React, { Component } from 'react';
-import Dropdown from 'react-dropdown';
 import 'react-dropdown/style.css';
-import {addStudent, getAllStudent} from "../services/studentService";
```

Comment:
Why deleting this import?  This causes an error when trying to run it. Is it caused by this branch is too far behind the main branch?